README.md

```diff
@@ -391,15 +391,171 @@ Ecosystem Platform.  Once UUIDs have been generated then
 products can be registered.  Once registered products and
 their artifacts can be viewed.
 
-### Generating UUIDs for a Data Product
+### Generating a Data Product
```

Comment:
Why are you duplicating the documentation you have in the product-generation.md file here? Just put a link to the correct readme, so you don't need to repeat yourself.

Reply:
removed duplicated documentation